tutorials/mnist.ipynb

```diff
@@ -0,0 +1,417 @@
+       " Your browser does not support the video tag.\n",
+       " </video>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Video object>"
+      ]
+     },
+     "execution_count": 155,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import Video\n",
+    "Video(\"../data/mnist/mnist.mp4\", embed=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "686adeb5",
+   "metadata": {},
+   "source": [
+    "### Load video into EVA"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b18cfe2a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Response Object:\n",
+      "@status: 0\n",
+      "@batch: Batch Object:\n",
+      "@dataframe:        Video\n",
+      "0  video.mp4\n",
+      "@batch_size: 1\n",
+      "@identifier_column: None\n",
+      "@metrics: None\n"
+     ]
+    }
+   ],
+   "source": [
+    "cursor.execute('LOAD DATA INFILE \"video.mp4\" INTO MyVideo;')\n",
+    "response = cursor.fetch_all()\n",
+    "print(response)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e93ee9e2",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'/Users/gtk/.eva/udfs/'"
+      ]
+     },
+     "execution_count": 82,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## Copy the example udf to the eva repository\n",
+    "from eva.configuration.dictionary import EVA_DEFAULT_DIR\n",
+    "import shutil\n",
+    "shutil.copytree('apps/', EVA_DEFAULT_DIR+ 'udfs/', dirs_exist_ok=True)"
```

Comment:
Do we still need this step? Since the below create uses the `tutorials/apps/mnist/eva_mnist_udf.py`.